### PR TITLE
Make onChangeArg more strict

### LINKED ISSUE #133: onChange not trigerred when calling clearSelection
`downshift` version: 1.0.0-rc.8

Hi,

Since the last RC release, I'm unable to clear the selection when using the controlled component.

The `onChange` function isn't trigerred anymore when calling `clearSelection`

### PR DESCRIPTION
**Pull request by jgoux**
**What**: Fix #133

**Why**: So the onChange function will be called when passing a null selectedItem

**How**: By making a check for onChange's argument more strict (null being falsy in JS)

**Checklist**:

- [ ] Documentation
- [ ] Tests
- [ ] Ready to be merged 
- [ ] Added myself to contributors table

### CODE REVIEW

File: src/downshift.js [hunk 1]
```diff
@@ -633,7 +633,7 @@ class Downshift extends Component {
     if (!uiDescriptor) {
       return null
     }
-    // doing React.Children.only for Preact support ⚛️
+    // doing React.Children.only for Preact support
```
Comment (kentcdodds) [inline src/downshift.js]: Awe 😕 why no emoji?

Comment (jgoux) [inline src/downshift.js]: Sorry as it was a really easy change I edited via Github directly, seems like the emoji goes away for some reason?

**Comment by kentcdodds** [comment]
This is great! Thanks! Do you think you could add a test so this doesn't happen again?

**Comment by kentcdodds** [comment]
Probably should go in the downshift.misc.js file

**Comment by kentcdodds** [comment]
```
~/Developer/downshift (master)
🏎  $ npm publish --tag rc
+ downshift@1.0.0-rc.9
```

**Pull request merged**